Tests/StarknetTests/Resources/TypedData/typed_data_rev_0_example.json

```diff
@@ -1,23 +1,47 @@
 {
   "types": {
-    "StarkNetDomain": [
-      { "name": "name", "type": "felt" },
-      { "name": "version", "type": "felt" },
-      { "name": "chainId", "type": "felt" }
+    "StarknetDomain": [
```

Comment:
Imo we shouldn't format these: it doesn't make those more readable and is unrelated to this PR.

Reply:
I've removed unnecessary formatting changes.